Adds release notes for private storage IR features (#70053)
openshift/openshift-docs: release_notes/ocp-4-15-release-notes.adoc

[id="ocp-4-15-registry"]
=== Registry

[id="ocp-4-15-private-storage-endpoint-azure"]
==== Support for private storage endpoint on Microsoft Azure

> With this release, the Image Registry Operator can be leveraged to use private storage endpoints on Microsoft Azure. This feature enables seamless configuration of private endpoints for storage accounts when {product-title} is deployed on private Azure clusters, which allows users to deploy the image registry without exposing public-facing storage endpoints.

Reviewer: Should "Microsoft Azure" be the attribute {azure-full}  ?

> With this release, the Image Registry Operator can be leveraged to use private storage endpoints on {azure-short}. This feature enables seamless configuration of private endpoints for storage accounts when {product-title} is deployed on private {azure-short} clusters, which allows users to deploy the image registry without exposing public-facing storage endpoints.

Reviewer 2: ```suggestion
With this release, the Image Registry Operator can be leveraged to use private storage endpoints on {azure-short}. You can use this feature to seamlessly configure private endpoints for storage accounts when {product-title} is deployed on private {azure-short} clusters, so that users can deploy the image registry without exposing public-facing storage endpoints.
```

Per IBMSG, avoid using "allow"
![image](https://github.com/openshift/openshift-docs/assets/40172997/cb5a68a6-ade1-4626-8b91-a91c404d9bf8)

Same thing for "enable"
![image](https://github.com/openshift/openshift-docs/assets/40172997/c1026456-940d-437f-856e-d61941a82d08)

Added some suggested wording to get around those.

> With this release, the Image Registry Operator can be leveraged to use private storage endpoints on {azure-short}. This feature enables seamless configuration of private endpoints for storage accounts when {product-title} is deployed on private {azure-short} clusters, which allows users to deploy the image registry without exposing public-facing storage endpoints.

Reviewer 2: Recommend using or adding an attribute for the Operator name

See xref:../post_installation_configuration/configuring-private-cluster.adoc#registry-configuring-private-storage-endpoint-azure_configuring-private-cluster[Configuring a private storage endpoint on Azure] for more information.

> For more information about installing a private image registry on a private Microsoft Azure cluster, see xref:../installing/installing_azure/installing-azure-private.adoc#installing-private-image-registry-private-azure[Optional: Preparing a private Microsoft Azure cluster for a private image registry].

Reviewer 2: Suggest maybe add an Additional resources section for the link since it's two at the end close together like this?

[id="ocp-4-15-registry"]
=== Registry

[id="ocp-4-15-private-storage-endpoint-azure"]
==== Support for private storage endpoint on Microsoft Azure

> With this release, the Image Registry Operator can be leveraged to use private storage endpoints on Microsoft Azure. This feature enables seamless configuration of private storage accounts when {product-title} is deployed on private Azure clusters, which allows users to deploy the image registry without exposing public-facing storage endpoints.

Reviewer 3: ```suggestion
With this release, the Image Registry Operator can be leveraged to use private storage endpoints on Microsoft Azure. This feature enables seamless configuration of private endpoints for storage accounts when {product-title} is deployed on private Azure clusters, which allows users to deploy the image registry without exposing public-facing storage endpoints.
```
